packages/webapp/src/stories/Pages/Finance/PureExpenseDetail.stories.jsx

```diff
@@ -30,7 +30,7 @@ const args = {
     console.log('Go back');
   },
   inputLabel: 'Custom Type Name',
-  customTypeRegister: 'custom_type_name',
+  nameFieldRegisterName: 'custom_type_name',
```

Comment:
Oh no did I write this?? 😳 

It's a sloppy copy-paste from the story below! There's no need for those two props in this component (I was like... why would Expense Detail need to be Edited here??!) Sorry!! Can you delete both `inputLabel` and `nameFieldRegisterName`?

Reply:
Good eye! I will check that out!

Reply:
gone!